semgrep/semgrep/output.py

```diff
@@ -209,7 +214,7 @@ def build_timing_summary(
 
     yield f"Slowest {items_to_show} rules to run (excluding parse time)"
     slowest_rule_times = sorted(
-        rule_timings.items(), key=lambda x: x[1][0], reverse=True
+        rule_timings.items(), key=lambda x: x[1][0], reverse=True  # type: ignore
```

Comment:
I'm not sure why this was necessary now and not before :confused:

Reply:
mypy confuses me :(